container/test-app/wc/clientAPI.html

```diff
@@ -33,9 +38,6 @@ <h3>
         ></luigi-container>
       </div>
 
-      <button id="init-button" style="margin: 25px 0 15px">
-        container.init() to initialize defered container
-      </button>
```

Comment:
should keep this button where it was. as it belongs to the container right below it. the one above doesnt relate to **init** scenario